flow360/component/simulation/operating_condition/operating_condition.py

```diff
@@ -401,21 +400,28 @@ def operating_condition_from_mach_reynolds(
     ...     alpha=2.0 * u.deg,
     ...     beta=0.0 * u.deg,
     ...     reference_mach=0.85,
+    ...     project_length_unit=1 * u.mm,
     ... )
     >>> print(condition)
     AerospaceCondition(...)
 
     """
 
-    thermal_state = ThermalState(
-        temperature=temperature,
-        material=Air(
-            dynamic_viscosity=Sutherland(
-                reference_temperature=temperature,
-                reference_viscosity=(mach / reynolds) * u.flow360_viscosity_unit,
-                effective_temperature=110.4 * u.K,
-            )
-        ),
+    if temperature == 288.15 * u.K:
+        log.info("Default value of 288.15 K will be used as temperature.")
```

Comment:
We should display a similar warning for default length unit.

Reply:
Or we should remove the default. Since the length unit appears in two places the default might cause a major error if it is used unintentionally.

Reply:
You are right, the user should have to specify the length unit, I will change it